desktop/components/fair_layout/templates/head.jade

```diff
@@ -1,59 +1,67 @@
-link( rel='apple-touch-icon', sizes='76x76', href=asset('/images/icon-76.png') )
+link( rel="canonical", href=(sd.APP_URL + sd.CURRENT_PATH) )
+link( rel='apple-touch-icon', href=asset('/images/icon-152.png') )
 link( rel='apple-touch-icon', sizes='120x120', href=asset('/images/icon-120.png') )
 link( rel='apple-touch-icon', sizes='152x152', href=asset('/images/icon-152.png') )
-link( rel='apple-touch-icon', href=asset('/images/icon-152.png') )
+link( rel='apple-touch-icon', sizes='76x76', href=asset('/images/icon-76.png') )
 link( rel='icon', type='image/png', href=asset('/images/favicon.ico') )
 link( rel='search', type='application/opensearchdescription+xml', href=asset('/images/opensearch.xml'), title='Artsy' )
-script( src=asset('/assets/scroll_frame.js') )
-title= fair.toPageTitle()
-meta( property="og:title", content=fair.toPageTitle())
+meta( charset='utf-8' )
+meta( name="apple-itunes-app" content="app-id=703796080, app-argument=#{sd.APP_URL}#{sd.CURRENT_PATH}" )
 meta( name="description", content=fair.toPageDescription() )
+meta( name='msapplication-config', content=asset('/images/browserconfig.xml') )
+meta( name='viewport', content='width=device-width, initial-scale=1, maximum-scale=1' )
+meta( property="al:ios:app_name" content='Artsy' )
+meta( property="al:ios:app_store_id" content='703796080' )
+meta( property="al:ios:url" content="#{sd.APP_URL}#{sd.CURRENT_PATH}" )
 meta( property="og:description", content=fair.toPageDescription() )
-meta( property="twitter:description", content=fair.toPageDescription())
-link( rel="canonical", href=(sd.APP_URL + sd.CURRENT_PATH) )
-meta( property="og:url", content=(sd.APP_URL + sd.CURRENT_PATH) )
-
-meta( property="og:type", content=(sd.FACEBOOK_APP_NAMESPACE + ":exhibition" ) )
-meta( property="og:start_time", content=fair.get('start_at'))
 meta( property="og:end_time", content=fair.get('end_at'))
-if fair.get('location')
-  meta( property="og:location", content=fair.get('location').raw )
+meta( property="og:start_time", content=fair.get('start_at'))
+meta( property="og:title", content=fair.toPageTitle())
+meta( property="og:type", content=(sd.FACEBOOK_APP_NAMESPACE + ":exhibition" ) )
+meta( property="og:url", content=(sd.APP_URL + sd.CURRENT_PATH) )
 meta( property="profile:type", content=profile.profileType() )
 meta( property="profile:username", content=profile.get('id') )
 meta( property="twitter:card", content="summary" )
+meta( property="twitter:description", content=fair.toPageDescription())
+script( src=asset('/assets/scroll_frame.js') )
+title= fair.toPageTitle()
+
+if fair.get('location')
+  meta( property="og:location", content=fair.get('location').raw )
+
 if profile.iconImageUrl()
   meta( property="og:image", content=profile.iconImageUrl() )
+
 if sd.INCLUDE_ESCAPED_FRAGMENT
   meta( name='fragment', content='!' )
-if sd.IS_RESPONSIVE && (userAgent && !userAgent.match('iPad'))
```

Comment:
Sorry for some of the noise in this PR. I had to fix these `viewport` meta tags that were using `sd.IS_RESPONSIVE` which is now removed. In the process I organized the rest of the template by grouping similar tags together and sorting alphabetically.